roles/telemetry_logging/tasks/logging_endpoint_tests.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Get Endpoint
+  ansible.builtin.shell:
+    cmd: |
+      oc project openstack     
+      kubectl exec openstackclient -- openstack endpoint list --service="{{ item[0] }}" --service="{{ item[1] }}"  --interface="{{ item[2] }}"
+  register: result 
+  changed_when: false
+  ignore_errors: yes
+
+- name: Set status "{{ item }}"
```

Comment:
This set of tasks is repeated across multiple files.
it is not necessary IMHO, since we have updates to the custom logger that will provide some reporting.

Please take a look at https://github.com/infrawatch/feature-verification-tests/pull/93 to see what output the custom_logger plugin produces. I recommend making adjustments there which will run to produce reports on the tests.

Reply:
Done